src/shared/migrations/041-add-more-slot-groups-attributes.ts

```diff
@@ -0,0 +1,33 @@
+import { DataTypes } from 'sequelize';
+
+/**
+ * Adds the requiredDLCs column to the MissionSlots table
+ */
+module.exports = {
+    up: async (queryInterface: any): Promise<void> => {
+        await queryInterface.addColumn('missionSlotGroups', 'radioFrequency', {
+            type: DataTypes.DOUBLE,
+            allowNull: true
+        });
+        await queryInterface.addColumn('missionSlotGroups', 'tacticalSymbol', {
+            type: DataTypes.ENUM,
+            values: ['inf'],
```

Comment:
Enum should have all possible values (as a list of constants, see [here](https://github.com/MorpheusXAUT/slotlist-backend/blob/master/src/shared/migrations/011-add-missions-visibility.ts) for an example) defined when added to table (at least all required up until that point).
Modifying an enum afterwards is a bit annoying, especially when default values are set.

Reply:
Yah, that was more a placeholder. I guess it makes sense to add all symbols that Arma knows about

Reply:
Yup, since we're ArmA focused anyways, just add whatever ArmA can handle (or the closest one to an international standard, like NATO). We can always modify the list later, but it's a bit of a hassle, so having all in at once would be much preferred 😄